3rdparty/openssl/CMakeLists.txt

```diff
@@ -80,6 +85,27 @@ ExternalProject_Add(
   BUILD_COMMAND ""
   INSTALL_COMMAND "")
 
+# The following commmand prepares the headers to be published. The headers
+# consist of those in the openssl/include/openssl folder, the opensslconf.h,
+# and the unsupported.h. The unsupported.h includes a list of APIs that are
+# disabled by OE for security concerns. Few headers are patched (using the
+# append-unsupported script) to include the unsupported.h so that OE can enforce
```

Comment:
```suggestion
# consist of those in the openssl/include/openssl folder, opensslconf.h,
# and unsupported.h. The unsupported.h header file includes a list of APIs that are
# disabled by OE for security concerns. A few headers are patched (using the
# append-unsupported script) to include unsupported.h so that OE can enforce
```